lib/notifications/client.rb

```diff
@@ -45,7 +45,7 @@ def send_sms(args)
     # @return [Notification]
     def get_notification(id)
       Notification.new(
-        speaker.send(:get, id)["notification"]
+        speaker.send(:get, id)["data"]["notification"]
       )
     end
```

Comment:
Is there a reason to call `#send(:get, id)`? Would `speaker.get(id)` work instead?

`send` is usually considered a bit dangerous and Rubyists tend to avoid it where they can. If you do have to use `send`, then `public_send` is preferable because it doesn't let you call private methods on the object.